README.md

```diff
@@ -199,6 +199,13 @@ build with doil (instances from version >=1.1) are able to be exported.
 
 See `doil pack:<command> --help` for more information
 
+To start the command `doil pack:import` with cron you should add the parameter -T or --no-term to the call. Otherwise you will get an error message that no TTY can be found.
```

Comment:
Why does this refer to `pack:import`, while the following example is for `pack:export`? And what is the difference between the two parameters? We should explain this better.

Reply:
Done!